tests/integration/features/bootstrap/FeatureContext.php

```diff
@@ -2960,6 +2961,44 @@ public function userDeleteTheAvatarOfRoom(string $user, string $identifier, int
 		$this->assertStatusCode($this->response, $statusCode);
 	}
 
+	/**
+	 * @When /^user "([^"]*)" starts "(invalid|audio|video)" recording in room "([^"]*)" with (\d+)(?: \((v1)\))?$/
+	 */
+	public function userStartRecordingInRoom(string $user, string $recordingType, string $identifier, int $statusCode, string $apiVersion = 'v1'): void {
+		$recordingTypes = [
+			'invalid' => -1,
+			'video' => 1,
+			'audio' => 2,
+		];
+
+		$data = [
+			'status' => $recordingTypes[$recordingType]
+		];
+
+		$this->setCurrentUser($user);
+		$roomToken = self::$identifierToToken[$identifier];
+		$this->sendRequest('POST', '/apps/spreed/api/' . $apiVersion . '/recording/' . $roomToken, $data);
+		$this->assertStatusCode($this->response, $statusCode);
+	}
+
+	/**
+	 * @When /^user "([^"]*)" stops recording in room "([^"]*)" with (\d+)(?: \((v1)\))?$/
+	 */
+	public function userStopRecordingInRoom(string $user, string $identifier, int $statusCode, string $apiVersion = 'v1'): void {
+		$this->setCurrentUser($user);
+		$roomToken = self::$identifierToToken[$identifier];
+		$this->sendRequest('DELETE', '/apps/spreed/api/' . $apiVersion . '/recording/' . $roomToken);
+		$this->assertStatusCode($this->response, $statusCode);
+	}
+
+	/**
+	 * @When the response json match with:
+	 */
+	public function theResponseJsonIsEqualTo(PyStringNode $jsonString): void {
+		$responseContent = $this->response->getBody()->getContents();
+		Assert::assertEquals((string) $jsonString, $responseContent, 'Invalid JSON string');
+	}
```

Comment:
```suggestion
	/**
	 * @Then the response error matches "([^"]*)"
	 */
	public function assertResponseErrorMatches(string $error): void {
		$responseData = $this->getDataFromResponse($this->response);
		Assert::assertEquals(['error' => $error], $responseData);
	}
```

Should work too and is much simpler as you don't have to create the full JSON thing around it?

Example snippet from test:
```
And the response error matches "recording"
```